### ISSUE TITLE
Hottrends parameters

### ISSUE THREAD

**Issue by burakuyar** [GeneralMills/pytrends#70]
Hi,

Firstly, many thanks for the library and updates :)

I'm trying to use hottrends function, by starting the example in the example.py. I have changed the country_payload to {'geo': 'TR'}. However, the results do not change when the country_payload changes. I have tried different countries and the result is always the same. Is there any points I'm missing or the parameters are not working as expected?

Cheers

**Comment by dreyco676** [comment]
If you go to the Google Trends page are you able to select a country for hottrends?

**Comment by burakuyar** [comment]
Yes, I can select the country on the right above the "Stories trending now" list. And, for example, when I select Turkey, the url becomes "https://www.google.com/trends/home/all/TR".

**Comment by magdalena90** [comment]
I have the same issue, hottrends returns a dictionary with numbers as keys and a list with 20 entries as its values

**Comment by magdalena90** [comment]
I think I figured out how to get the correct result, the dictionary the api returns includes all the countries' hot trends, and using as key the number corresponding to your country (which I got when I manually entered the google trends page, selected the country I wanted to examine and saw the parameter in the url), you get the hot trends of that country.

For example, the google trends url for Mexico is https://www.google.com/trends/hottrends#pn=p21, so by doing hotTrends['21'] I got the corresponding hot trends
